core/chains/evm/chain_set.go

```diff
@@ -73,7 +73,7 @@ func (cll *chainSet) Start(ctx context.Context) error {
 	var ms services.MultiStart
 	for _, c := range cll.Chains() {
 		if err := ms.Start(ctx, c); err != nil {
-			return errors.Wrapf(err, "failed to start chain %q", c.ID())
+			return errors.Wrapf(err, "failed to start chain %s", c.ID().String())
```

Comment:
I think these were quoted in case there's spaces or weird characters. Maybe worth preserving the separation somehow:
```suggestion
			return errors.Wrapf(err, "failed to start chain '%s'", c.ID().String())
```
```suggestion
			return errors.Wrapf(err, "failed to start chain: %s", c.ID().String())
```

Reply:
ah i misread the spec - realized i was reading it as `%q` was only valid for numbers.

i reduced the changes to just to use `ID().String()`

Reply:
it used to use the string method previously: https://github.com/smartcontractkit/chainlink/blame/d00393223659cf6eb18c123bbc2572bd1e6348e6/core/chains/evm/chain_set.go#LL82C2-L82C2